app-libs/stf/src/helpers.rs

```diff
@@ -69,7 +69,7 @@ pub fn get_storage_by_key_hash<V: Decode>(key: Vec<u8>) -> Option<V> {
 			None
 		}
 	} else {
-		error!("key not found in state {:x?}", key);
+		info!("key not found in state {:x?}", key);
```

Comment:
Hm. That's not an error because it's possible to request not yet set state?

Reply:
It happens regularly, when e.g. when an account does not exist yet. (that is detected and a new account is then created) That's why I changed it to info, because it is part of the expected behavior.